tasks/sanity-inspect-image.yaml

```diff
@@ -20,9 +20,9 @@ spec:
     image: quay.io/redhat-appstudio/hacbs-test:latest
     workingDir: $(workspaces.workspace.path)/hacbs/$(context.task.name)
     script: |
-      IMAGE_INSPECT=image_inspect.json
-      BASE_IMAGE_INSPECT=base_image_inspect.json
-      RAW_IMAGE_INSPECT=raw_image_inspect.json
+      IMAGE_INSPECT=$(workspaces.workspace.path)/image_inspect.json
+      BASE_IMAGE_INSPECT=$(workspaces.workspace.path)/base_image_inspect.json
+      RAW_IMAGE_INSPECT=$(workspaces.workspace.path)/raw_image_inspect.json
```

Comment:
adding files into root dir of the workspace is not good idea. Hacbs things should not be mixed with source code.